site/content/blog/2021-10-01-whats-new-in-svelte-october-2021.md

```diff
@@ -0,0 +1,97 @@
+---
+title: What's new in Svelte: October 2021
+description: A whole year of "What's new in Svelte"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Hey y'all 👋 It's been 1 year since "What's new in Svelte" started being cross-posted to the Svelte blog. I wanted to take this moment at the top to thank all of you for reading and for all the contributors every month. From the maintainers to everyone who posts their work in the Discord and Reddit, it's amazing to witness all the effort that goes into making the Svelte community great.
+
+Keep up the good work, everyone! Now, let's dive into this month's news...
+
+## New around Svelte
+
+- New additions to Svelte's export map now expose no-op versions of lifecycle functions for SSR. This prevents the need for bundlers to bundle external dependencies that imports `svelte` in order to resolve them to `svelte/ssr` (Svelte **3.43.0**)
+- Custom components with a `src` attribute no longer break `svelte-native` builds (Svelte **3.42.4**)
+- Svelte plugin users without [the TypeScript plugin](https://github.com/sveltejs/language-tools/issues/580#issuecomment-832502069) enabled will now be prompted to enable it (Svelte extensions **105.4.0**)
+- Event modifiers have been added to intellisense as autocomplete and hover info (Svelte extensions **105.4.0**)
+
+For a full list of features and bug fixes, check out the [Svelte changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## SvelteKit Updates
+
+Svelte maintainers are [looking for help getting SvelteKit to 1.0](https://github.com/sveltejs/kit/issues/2100). Antony said it well in a [recent comment](https://github.com/sveltejs/kit/issues/2100#issuecomment-895446285) on the issue: 
+
+> If you think you are too n00b to contribute (you're not), then add tests, or write tests for the feature you want to add, before you add it! Start small, and learn the codebase that way.
+
+If you'd like to help, please consider working on any of the [1.0 milestone issues](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0).
+
+Notable SvelteKit improvements this month include...
+
+- Service workers are now allowed to access files using the \$lib alias ([#2326](https://github.com/sveltejs/kit/pull/2326))
+- Nested dependency optimization has now been enabled with vite ([#2343](https://github.com/sveltejs/kit/pull/2343))
```

Comment:
```suggestion
- Nested dependency optimization has now been enabled with Vite ([#2343](https://github.com/sveltejs/kit/pull/2343))
```

Reply:
I'm also not sure if we want to keep this phrase. Maybe something like:
```suggestion
- Svelte libraries should now work out-of-the-box without any Vite configuration ([#2343](https://github.com/sveltejs/kit/pull/2343))
```